### PR TITLE
Principles titling

### PR DESCRIPTION
**Pull request by cwilso**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/AB-public/pull/124.html" title="Last updated on Sep 12, 2023, 4:19 PM UTC (7a3ec4f)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/AB-public/124/b3380a1...7a3ec4f.html" title="Last updated on Sep 12, 2023, 4:19 PM UTC (7a3ec4f)">Diff</a>

### CODE REVIEW

**Comment by cwilso** [comment]
Fixes #114

File: Vision/Vision.bs [hunk 1]
```diff
@@ -142,34 +143,34 @@ Boilerplate: omit conformance
...
 		collaborating on innovations across our community.
-	* We aim to reduce centralization in Web architecture,
+	* **Decentralization**: We aim to reduce centralization in Web architecture,
 		minimizing single points of failure
 		and single points of control.
-	* We are committed to establishing and improving collaborative relationships
+	* **Collaboration**: We are committed to establishing and improving collaborative relationships
 		with other Internet and Web standards organizations,
 		and building and maintaining respected relationships
 		with governments and businesses for providing credible advice.
```
Comment (wareid) [inline Vision/Vision.bs:176]: It's too late in the day for me to remember the grammatical term for this but I wonder if this is more consistent if each principle is an active not passive term.

Diversity >> Diverse
Consensus >> Consensus-Driven
Interoperability >> Interoperable 
Decentralization >> Decentralized
Collaboration >> Collaborative

Comment (wareid) [reply Vision/Vision.bs:176]: Otherwise, this is exactly what I was looking for!

**Review by tantek — APPROVED** [review]
This principles labels look good to me, so thumbs up on this PR.

However the labels when read as a list imply a sort of ordering which we may want to consider and sort accordingly. I agree with Users-first being first.

File: Vision/Vision.bs [hunk 2]
```diff
@@ -142,34 +143,34 @@ Boilerplate: omit conformance
 		and more.
 		In order to ensure the W3C serves the needs of the entire Web user base, 
 		we also strive to broaden diversity and inclusion for our own participants.
-	* We create standards with an ethical intent to improve 
+	* **Horizontal Review**: We create standards with an ethical intent to improve 
 		equity of access and participation on the Web.
```
Comment (dwsinger) [inline Vision/Vision.bs:147]: Access Equity is a better summary and is in the text

File: Vision/Vision.bs [hunk 3]
```diff
@@ -142,34 +143,34 @@ Boilerplate: omit conformance
 		and more.
 		In order to ensure the W3C serves the needs of the entire Web user base, 
 		we also strive to broaden diversity and inclusion for our own participants.
-	* We create standards with an ethical intent to improve 
+	* **Horizontal Review**: We create standards with an ethical intent to improve 
 		equity of access and participation on the Web. 
 		We will ensure the technical standards of the Web meets broad goals 
 		using consistent horizontal review to ensure accessibility,
```
Comment (dwsinger) [inline Vision/Vision.bs:149]: delete "horizontal", it's a buzz word

File: Vision/Vision.bs (hunk 2 above)
Comment (fantasai) [reply Vision/Vision.bs:147]: This needs bikeshedding. we should do that in https://github.com/w3c/AB-public/pull/122/files after this lands.

**Pull request merged**